base/bitarray.jl

```diff
@@ -431,7 +431,7 @@ end
 
 function copyto!(dest::BitArray, src::Array)
     length(src) > length(dest) && throw(BoundsError(dest, length(dest)+1))
-    length(src) == 0 && return det
+    length(src) == 0 && return dest
```

Comment:
sneaky... I will split this out and make a separate PR.

Reply:
```
@code_warntype copyto!(BitArray([true]), [1])
....
 end::Union{typeof(Base.det), BitArray{1}}
```

lol...

Reply:
I now understand how Jameson found this:
```
WARNING: Base.det is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add a `using LinearAlgebra` to your imports..
 in module Base
in copyto! at bitarray.jl
```